tests/test_raid.py

```diff
@@ -61,7 +62,7 @@ def test_raid_error(self, run_command, fs):
         assert e.value.params[1] == 'Error!'
 
     def test_raid_missing(self, run_command, fs):
-        run_command.side_effect = subprocess.CalledProcessError(1, MDADM_DETAIL_CMD, None)
+        run_command.side_effect = ProcessException(1, MDADM_DETAIL_CMD, "", "")
```

Comment:
Would testing with non empty stderr and stdout be safer? (strings can be empty by accident, but are less likely to match a specific phrase by accident)